client/attest.go

```diff
@@ -62,5 +98,14 @@ func (k *Key) Attest(opts AttestOpts) (*pb.Attestation, error) {
 	if len(opts.CanonicalEventLog) != 0 {
 		attestation.CanonicalEventLog = opts.CanonicalEventLog
 	}
+
+	// Construct certficate chain.
+	if opts.ValidateCertChain {
```

Comment:
We should also check that the key even has a cert (otherwise I think we will have a nil dereference).